Data/Gen3sis_parameter_tables/M1/North_America_parameters.txt
Sim_index	S	omega_T	omega_P	sigma_e	disp_shape	disp_scale	seed	divergence_factor	max_species	max_coex_sp	start_species	init_ab	grid_cell_distance
Simulation_1	1.998	0.055	0.175	0.078	1.75	0.549	1	0.05	12000	1000	1	500	111
Simulation_2	2.501	0.032	0.111	0.041	2.377	0.323	2	0.05	12000	1000	1	500	111
Simulation_3	1.495	0.078	0.239	0.114	1.123	0.775	3	0.05	12000	1000	1	500	111
Simulation_4	1.747	0.044	0.144	0.023	2.691	0.889	4	0.05	12000	1000	1	500	111
Simulation_5	2.752	0.089	0.269	0.096	1.436	0.436	5	0.05	12000	1000	1	500	111
Simulation_6	2.25	0.021	0.081	0.059	0.809	0.662	6	0.05	12000	1000	1	500	111
Simulation_7	1.244	0.066	0.206	0.132	2.064	0.21	7	0.05	12000	1000	1	500	111
Simulation_8	1.369	0.038	0.128	0.105	1.907	0.266	8	0.05	12000	1000	1	500	111
Simulation_9	2.375	0.083	0.253	0.032	0.652	0.719	9	0.05	12000	1000	1	500	111
Simulation_10	2.878	0.015	0.064	0.141	1.279	0.493	10	0.05	12000	1000	1	500	111
Simulation_11	1.872	0.061	0.192	0.068	2.534	0.945	11	0.05	12000	1000	1	500	111
Simulation_12	1.621	0.027	0.097	0.087	1.593	0.832	12	0.05	12000	1000	1	500	111
Simulation_13	2.627	0.072	0.222	0.014	2.848	0.379	13	0.05	12000	1000	1	500	111
Simulation_14	2.124	0.049	0.158	0.123	2.221	0.606	14	0.05	12000	1000	1	500	111
Simulation_15	1.118	0.095	0.286	0.05	0.966	0.153	15	0.05	12000	1000	1	500	111
Simulation_16	1.181	0.052	0.167	0.064	1.201	0.408	16	0.05	12000	1000	1	500	111
Simulation_17	2.187	0.097	0.292	0.137	2.456	0.86	17	0.05	12000	1000	1	500	111
Simulation_18	2.69	0.029	0.103	0.027	1.828	0.181	18	0.05	12000	1000	1	500	111
Simulation_19	1.684	0.075	0.231	0.1	0.574	0.634	19	0.05	12000	1000	1	500	111
Simulation_20	1.935	0.018	0.072	0.046	2.142	0.747	20	0.05	12000	1000	1	500	111
Simulation_21	2.941	0.063	0.197	0.118	0.887	0.294	21	0.05	12000	1000	1	500	111
Simulation_22	2.438	0.041	0.136	0.009	1.515	0.973	22	0.05	12000	1000	1	500	111
Simulation_23	1.432	0.086	0.261	0.082	2.77	0.521	23	0.05	12000	1000	1	500	111
Simulation_24	1.306	0.024	0.089	0.128	2.613	0.464	24	0.05	12000	1000	1	500	111
Simulation_25	2.312	0.069	0.214	0.055	1.358	0.917	25	0.05	12000	1000	1	500	111
Simulation_26	2.815	0.046	0.15	0.091	0.73	0.238	26	0.05	12000	1000	1	500	111
Simulation_27	1.809	0.092	0.278	0.018	1.985	0.691	27	0.05	12000	1000	1	500	111
Simulation_28	1.558	0.035	0.119	0.146	1.044	0.577	28	0.05	12000	1000	1	500	111
Simulation_29	2.564	0.08	0.244	0.073	2.299	0.125	29	0.05	12000	1000	1	500	111
Simulation_30	2.061	0.012	0.056	0.109	2.926	0.804	30	0.05	12000	1000	1	500	111
Simulation_31	1.055	0.058	0.183	0.037	1.672	0.351	31	0.05	12000	1000	1	500	111
Simulation_32	1.086	0.034	0.117	0.089	2.26	0.676	32	0.05	12000	1000	1	500	111
Simulation_33	2.092	0.079	0.242	0.016	1.005	0.224	33	0.05	12000	1000	1	500	111
Simulation_34	2.595	0.011	0.053	0.125	1.632	0.903	34	0.05	12000	1000	1	500	111
Simulation_35	1.589	0.056	0.178	0.052	2.887	0.45	35	0.05	12000	1000	1	500	111
Simulation_36	1.841	0.022	0.083	0.107	1.319	0.337	36	0.05	12000	1000	1	500	111
Simulation_37	2.847	0.068	0.211	0.034	2.574	0.79	37	0.05	12000	1000	1	500	111
Simulation_38	2.344	0.045	0.147	0.143	1.946	0.111	38	0.05	12000	1000	1	500	111
Simulation_39	1.338	0.09	0.272	0.071	0.691	0.563	39	0.05	12000	1000	1	500	111
Simulation_40	1.464	0.017	0.069	0.025	0.848	0.62	40	0.05	12000	1000	1	500	111
Simulation_41	2.47	0.062	0.194	0.098	2.103	0.167	41	0.05	12000	1000	1	500	111
Simulation_42	2.972	0.039	0.131	0.062	2.73	0.846	42	0.05	12000	1000	1	500	111
Simulation_43	1.967	0.085	0.258	0.134	1.475	0.394	43	0.05	12000	1000	1	500	111
Simulation_44	1.715	0.051	0.164	0.007	2.417	0.507	44	0.05	12000	1000	1	500	111
Simulation_45	2.721	0.096	0.289	0.08	1.162	0.959	45	0.05	12000	1000	1	500	111
Simulation_46	2.218	0.028	0.1	0.043	0.534	0.28	46	0.05	12000	1000	1	500	111
Simulation_47	1.212	0.073	0.225	0.116	1.789	0.733	47	0.05	12000	1000	1	500	111
Simulation_48	1.149	0.031	0.108	0.148	2.966	0.988	48	0.05	12000	1000	1	500	111
Simulation_49	2.155	0.076	0.233	0.075	1.711	0.535	49	0.05	12000	1000	1	500	111
Simulation_50	2.658	0.053	0.169	0.112	1.083	0.761	50	0.05	12000	1000	1	500	111
Simulation_51	1.652	0.099	0.297	0.039	2.338	0.309	51	0.05	12000	1000	1	500	111
Simulation_52	1.904	0.042	0.139	0.13	0.77	0.195	52	0.05	12000	1000	1	500	111
Simulation_53	2.91	0.087	0.264	0.057	2.025	0.648	53	0.05	12000	1000	1	500	111
Simulation_54	2.407	0.02	0.078	0.093	2.652	0.422	54	0.05	12000	1000	1	500	111
Simulation_55	1.401	0.065	0.203	0.021	1.397	0.874	55	0.05	12000	1000	1	500	111
Simulation_56	1.275	0.048	0.156	0.048	1.554	0.818	56	0.05	12000	1000	1	500	111
Simulation_57	2.281	0.093	0.281	0.121	2.809	0.365	57	0.05	12000	1000	1	500	111
Simulation_58	2.784	0.025	0.092	0.012	2.181	0.592	58	0.05	12000	1000	1	500	111
Simulation_59	1.778	0.07	0.217	0.084	0.926	0.139	59	0.05	12000	1000	1	500	111
Simulation_60	1.527	0.014	0.061	0.066	1.868	0.252	60	0.05	12000	1000	1	500	111
Simulation_61	2.532	0.059	0.186	0.139	0.613	0.705	61	0.05	12000	1000	1	500	111
Simulation_62	2.029	0.037	0.125	0.03	1.24	0.478	62	0.05	12000	1000	1	500	111
Simulation_63	1.024	0.082	0.25	0.103	2.495	0.931	63	0.05	12000	1000	1	500	111
Simulation_64	1.039	0.046	0.15	0.122	1.025	0.882	64	0.05	12000	1000	1	500	111
Simulation_65	2.045	0.091	0.275	0.049	2.279	0.429	65	0.05	12000	1000	1	500	111
Simulation_66	2.548	0.023	0.086	0.085	2.907	0.655	66	0.05	12000	1000	1	500	111
Simulation_67	1.542	0.068	0.211	0.013	1.652	0.203	67	0.05	12000	1000	1	500	111
Simulation_68	1.794	0.012	0.056	0.14	2.593	0.316	68	0.05	12000	1000	1	500	111
Simulation_69	2.8	0.057	0.181	0.067	1.338	0.768	69	0.05	12000	1000	1	500	111
Simulation_70	2.297	0.034	0.117	0.104	0.711	0.542	70	0.05	12000	1000	1	500	111
Simulation_71	1.291	0.08	0.244	0.031	1.966	0.995	71	0.05	12000	1000	1	500	111
Simulation_72	1.417	0.029	0.103	0.076	2.123	0.938	72	0.05	12000	1000	1	500	111
Simulation_73	2.422	0.074	0.228	0.149	0.868	0.485	73	0.05	12000	1000	1	500	111
Simulation_74	2.925	0.051	0.164	0.04	1.495	0.712	74	0.05	12000	1000	1	500	111
Simulation_75	1.919	0.097	0.292	0.113	2.75	0.259	75	0.05	12000	1000	1	500	111
Simulation_76	1.668	0.04	0.133	0.058	1.181	0.146	76	0.05	12000	1000	1	500	111
Simulation_77	2.674	0.085	0.258	0.131	2.436	0.599	77	0.05	12000	1000	1	500	111
Simulation_78	2.171	0.017	0.069	0.022	1.809	0.372	78	0.05	12000	1000	1	500	111
Simulation_79	1.165	0.063	0.197	0.095	0.554	0.825	79	0.05	12000	1000	1	500	111
Simulation_80	1.228	0.02	0.078	0.099	1.73	0.57	80	0.05	12000	1000	1	500	111
Simulation_81	2.234	0.066	0.206	0.026	2.985	0.118	81	0.05	12000	1000	1	500	111
Simulation_82	2.737	0.043	0.142	0.136	2.358	0.797	82	0.05	12000	1000	1	500	111
Simulation_83	1.731	0.088	0.267	0.063	1.103	0.344	83	0.05	12000	1000	1	500	111
Simulation_84	1.982	0.054	0.172	0.081	2.044	0.457	84	0.05	12000	1000	1	500	111
Simulation_85	2.988	0.099	0.297	0.008	0.789	0.91	85	0.05	12000	1000	1	500	111
Simulation_86	2.485	0.032	0.111	0.117	1.417	0.231	86	0.05	12000	1000	1	500	111
Simulation_87	1.479	0.077	0.236	0.045	2.672	0.683	87	0.05	12000	1000	1	500	111
Simulation_88	1.354	0.037	0.125	0.017	2.828	0.74	88	0.05	12000	1000	1	500	111
Simulation_89	2.36	0.082	0.25	0.09	1.574	0.287	89	0.05	12000	1000	1	500	111
Simulation_90	2.862	0.015	0.064	0.054	0.946	0.966	90	0.05	12000	1000	1	500	111
Simulation_91	1.857	0.06	0.189	0.126	2.201	0.514	91	0.05	12000	1000	1	500	111
Simulation_92	1.605	0.026	0.094	0.035	0.632	0.401	92	0.05	12000	1000	1	500	111
Simulation_93	2.611	0.071	0.219	0.108	1.887	0.853	93	0.05	12000	1000	1	500	111
Simulation_94	2.108	0.049	0.158	0.072	2.515	0.174	94	0.05	12000	1000	1	500	111
Simulation_95	1.102	0.094	0.283	0.145	1.26	0.627	95	0.05	12000	1000	1	500	111
Simulation_96	1.071	0.022	0.083	0.056	1.77	0.528	96	0.05	12000	1000	1	500	111
Simulation_97	2.077	0.067	0.208	0.129	0.515	0.981	97	0.05	12000	1000	1	500	111
Simulation_98	2.58	0.044	0.144	0.02	1.142	0.302	98	0.05	12000	1000	1	500	111
Simulation_99	1.574	0.09	0.272	0.092	2.397	0.754	99	0.05	12000	1000	1	500	111
Simulation_100	1.825	0.033	0.114	0.074	1.456	0.641	100	0.05	12000	1000	1	500	111
Simulation_101	2.831	0.078	0.239	0.147	2.711	0.188	101	0.05	12000	1000	1	500	111
Simulation_102	2.328	0.01	0.05	0.038	2.083	0.867	102	0.05	12000	1000	1	500	111
Simulation_103	1.322	0.056	0.178	0.11	0.828	0.415	103	0.05	12000	1000	1	500	111
Simulation_104	1.448	0.05	0.161	0.138	0.672	0.358	104	0.05	12000	1000	1	500	111
Simulation_105	2.454	0.095	0.286	0.065	1.926	0.811	105	0.05	12000	1000	1	500	111
Simulation_106	2.957	0.027	0.097	0.101	2.554	0.132	106	0.05	12000	1000	1	500	111
Simulation_107	1.951	0.073	0.225	0.029	1.299	0.584	107	0.05	12000	1000	1	500	111
Simulation_108	1.699	0.016	0.067	0.12	2.868	0.698	108	0.05	12000	1000	1	500	111
Simulation_109	2.705	0.061	0.192	0.047	1.613	0.245	109	0.05	12000	1000	1	500	111
Simulation_110	2.202	0.039	0.131	0.083	0.985	0.924	110	0.05	12000	1000	1	500	111
Simulation_111	1.196	0.084	0.256	0.01	2.24	0.471	111	0.05	12000	1000	1	500	111
Simulation_112	1.134	0.041	0.136	0.033	2.475	0.217	112	0.05	12000	1000	1	500	111
Simulation_113	2.139	0.087	0.264	0.106	1.221	0.669	113	0.05	12000	1000	1	500	111
Simulation_114	2.642	0.019	0.075	0.07	0.593	0.443	114	0.05	12000	1000	1	500	111
Simulation_115	1.637	0.064	0.2	0.142	1.848	0.896	115	0.05	12000	1000	1	500	111
Simulation_116	1.888	0.03	0.106	0.015	0.907	0.783	116	0.05	12000	1000	1	500	111
Simulation_117	2.894	0.075	0.231	0.088	2.162	0.33	117	0.05	12000	1000	1	500	111
Simulation_118	2.391	0.053	0.169	0.051	2.789	0.556	118	0.05	12000	1000	1	500	111
Simulation_119	1.385	0.098	0.294	0.124	1.534	0.104	119	0.05	12000	1000	1	500	111
Simulation_120	1.259	0.013	0.058	0.079	1.377	0.16	120	0.05	12000	1000	1	500	111
Simulation_121	2.265	0.058	0.183	0.006	2.632	0.613	121	0.05	12000	1000	1	500	111
Simulation_122	2.768	0.036	0.122	0.115	2.005	0.386	122	0.05	12000	1000	1	500	111
Simulation_123	1.762	0.081	0.247	0.042	0.75	0.839	123	0.05	12000	1000	1	500	111
Simulation_124	1.511	0.047	0.153	0.097	2.319	0.952	124	0.05	12000	1000	1	500	111
Simulation_125	2.517	0.092	0.278	0.024	1.064	0.5	125	0.05	12000	1000	1	500	111
Simulation_126	2.014	0.024	0.089	0.133	1.691	0.726	126	0.05	12000	1000	1	500	111
Simulation_127	1.008	0.07	0.217	0.06	2.946	0.273	127	0.05	12000	1000	1	500	111
Simulation_128	1.016	0.04	0.133	0.142	1.662	0.475	128	0.05	12000	1000	1	500	111
Simulation_129	2.022	0.085	0.258	0.069	2.917	0.928	129	0.05	12000	1000	1	500	111
Simulation_130	2.525	0.017	0.069	0.105	2.289	0.249	130	0.05	12000	1000	1	500	111
Simulation_131	1.519	0.062	0.194	0.033	1.034	0.701	131	0.05	12000	1000	1	500	111
Simulation_132	1.77	0.028	0.1	0.124	1.975	0.588	132	0.05	12000	1000	1	500	111
Simulation_133	2.776	0.074	0.228	0.051	0.721	0.135	133	0.05	12000	1000	1	500	111
Simulation_134	2.273	0.051	0.164	0.087	1.348	0.814	134	0.05	12000	1000	1	500	111
Simulation_135	1.267	0.096	0.289	0.014	2.603	0.362	135	0.05	12000	1000	1	500	111
Simulation_136	1.393	0.011	0.053	0.042	2.76	0.418	136	0.05	12000	1000	1	500	111
Simulation_137	2.399	0.057	0.181	0.114	1.505	0.871	137	0.05	12000	1000	1	500	111
Simulation_138	2.902	0.034	0.117	0.005	0.877	0.192	138	0.05	12000	1000	1	500	111
Simulation_139	1.896	0.079	0.242	0.078	2.132	0.645	139	0.05	12000	1000	1	500	111
Simulation_140	1.644	0.045	0.147	0.06	0.564	0.758	140	0.05	12000	1000	1	500	111
Simulation_141	2.65	0.091	0.275	0.133	1.819	0.305	141	0.05	12000	1000	1	500	111
Simulation_142	2.147	0.023	0.086	0.023	2.446	0.984	142	0.05	12000	1000	1	500	111
Simulation_143	1.141	0.068	0.211	0.096	1.191	0.531	143	0.05	12000	1000	1	500	111
Simulation_144	1.204	0.026	0.094	0.092	1.113	0.277	144	0.05	12000	1000	1	500	111
Simulation_145	2.21	0.071	0.219	0.019	2.368	0.729	145	0.05	12000	1000	1	500	111
Simulation_146	2.713	0.048	0.156	0.128	2.995	0.503	146	0.05	12000	1000	1	500	111
Simulation_147	1.707	0.093	0.281	0.055	1.74	0.956	147	0.05	12000	1000	1	500	111
Simulation_148	1.959	0.037	0.125	0.11	2.681	0.843	148	0.05	12000	1000	1	500	111
Simulation_149	2.965	0.082	0.25	0.037	1.426	0.39	149	0.05	12000	1000	1	500	111
Simulation_150	2.462	0.014	0.061	0.146	0.799	0.616	150	0.05	12000	1000	1	500	111
Simulation_151	1.456	0.06	0.189	0.074	2.054	0.164	151	0.05	12000	1000	1	500	111
Simulation_152	1.33	0.054	0.172	0.028	2.211	0.107	152	0.05	12000	1000	1	500	111
Simulation_153	2.336	0.099	0.297	0.101	0.956	0.56	153	0.05	12000	1000	1	500	111
Simulation_154	2.839	0.031	0.108	0.064	1.583	0.333	154	0.05	12000	1000	1	500	111
Simulation_155	1.833	0.076	0.233	0.137	2.838	0.786	155	0.05	12000	1000	1	500	111
Simulation_156	1.582	0.02	0.078	0.01	1.27	0.899	156	0.05	12000	1000	1	500	111
Simulation_157	2.587	0.065	0.203	0.083	2.525	0.447	157	0.05	12000	1000	1	500	111
Simulation_158	2.084	0.043	0.142	0.046	1.897	0.673	158	0.05	12000	1000	1	500	111
Simulation_159	1.079	0.088	0.267	0.119	0.642	0.22	159	0.05	12000	1000	1	500	111
Simulation_160	1.11	0.016	0.067	0.062	2.407	0.828	160	0.05	12000	1000	1	500	111
Simulation_161	2.116	0.061	0.192	0.135	1.152	0.376	161	0.05	12000	1000	1	500	111
Simulation_162	2.619	0.038	0.128	0.026	0.525	0.602	162	0.05	12000	1000	1	500	111
Simulation_163	1.613	0.084	0.256	0.099	1.779	0.15	163	0.05	12000	1000	1	500	111
Simulation_164	1.864	0.05	0.161	0.044	0.838	0.263	164	0.05	12000	1000	1	500	111
Simulation_165	2.87	0.095	0.286	0.117	2.093	0.715	165	0.05	12000	1000	1	500	111
Simulation_166	2.367	0.027	0.097	0.008	2.721	0.489	166	0.05	12000	1000	1	500	111
Simulation_167	1.361	0.072	0.222	0.08	1.466	0.942	167	0.05	12000	1000	1	500	111
Simulation_168	1.487	0.033	0.114	0.126	1.309	0.998	168	0.05	12000	1000	1	500	111
Simulation_169	2.493	0.078	0.239	0.053	2.564	0.546	169	0.05	12000	1000	1	500	111
Simulation_170	2.996	0.01	0.05	0.089	1.936	0.772	170	0.05	12000	1000	1	500	111
Simulation_171	1.99	0.055	0.175	0.017	0.681	0.319	171	0.05	12000	1000	1	500	111
Simulation_172	1.739	0.021	0.081	0.144	2.25	0.206	172	0.05	12000	1000	1	500	111
Simulation_173	2.745	0.067	0.208	0.071	0.995	0.659	173	0.05	12000	1000	1	500	111
Simulation_174	2.242	0.044	0.144	0.108	1.623	0.432	174	0.05	12000	1000	1	500	111
Simulation_175	1.236	0.089	0.269	0.035	2.877	0.885	175	0.05	12000	1000	1	500	111
Simulation_176	1.173	0.047	0.153	0.012	1.858	0.63	176	0.05	12000	1000	1	500	111
Simulation_177	2.179	0.092	0.278	0.085	0.603	0.178	177	0.05	12000	1000	1	500	111
Simulation_178	2.682	0.024	0.089	0.048	1.23	0.857	178	0.05	12000	1000	1	500	111
Simulation_179	1.676	0.069	0.214	0.121	2.485	0.404	179	0.05	12000	1000	1	500	111
Simulation_180	1.927	0.013	0.058	0.03	1.544	0.517	180	0.05	12000	1000	1	500	111
Simulation_181	2.933	0.058	0.183	0.103	2.799	0.97	181	0.05	12000	1000	1	500	111
Simulation_182	2.43	0.035	0.119	0.067	2.172	0.291	182	0.05	12000	1000	1	500	111
Simulation_183	1.424	0.081	0.247	0.139	0.917	0.744	183	0.05	12000	1000	1	500	111
Simulation_184	1.299	0.03	0.106	0.112	0.76	0.687	184	0.05	12000	1000	1	500	111
Simulation_185	2.305	0.075	0.231	0.039	2.015	0.234	185	0.05	12000	1000	1	500	111
Simulation_186	2.807	0.052	0.167	0.149	2.642	0.913	186	0.05	12000	1000	1	500	111
Simulation_187	1.802	0.098	0.294	0.076	1.387	0.461	187	0.05	12000	1000	1	500	111
Simulation_188	1.55	0.041	0.136	0.094	2.956	0.348	188	0.05	12000	1000	1	500	111
Simulation_189	2.556	0.086	0.261	0.021	1.701	0.8	189	0.05	12000	1000	1	500	111
Simulation_190	2.053	0.018	0.072	0.13	1.074	0.121	190	0.05	12000	1000	1	500	111
Simulation_191	1.047	0.064	0.2	0.058	2.328	0.574	191	0.05	12000	1000	1	500	111
Simulation_192	1.031	0.028	0.1	0.041	1.132	0.737	192	0.05	12000	1000	1	500	111
Simulation_193	2.037	0.073	0.225	0.113	2.387	0.284	193	0.05	12000	1000	1	500	111
Simulation_194	2.54	0.05	0.161	0.077	1.76	0.963	194	0.05	12000	1000	1	500	111
Simulation_195	1.534	0.096	0.289	0.15	0.505	0.51	195	0.05	12000	1000	1	500	111
Simulation_196	1.786	0.039	0.131	0.022	2.074	0.397	196	0.05	12000	1000	1	500	111
Simulation_197	2.792	0.084	0.256	0.095	0.819	0.85	197	0.05	12000	1000	1	500	111
Simulation_198	2.289	0.016	0.067	0.059	1.446	0.171	198	0.05	12000	1000	1	500	111
Simulation_199	1.283	0.062	0.194	0.132	2.701	0.623	199	0.05	12000	1000	1	500	111
Simulation_200	1.409	0.045	0.147	0.086	2.544	0.567	200	0.05	12000	1000	1	500	111
Simulation_201	2.415	0.09	0.272	0.013	1.289	0.114	201	0.05	12000	1000	1	500	111
Simulation_202	2.917	0.022	0.083	0.122	0.662	0.793	202	0.05	12000	1000	1	500	111
Simulation_203	1.912	0.067	0.208	0.05	1.917	0.34	203	0.05	12000	1000	1	500	111
Simulation_204	1.66	0.011	0.053	0.104	0.975	0.454	204	0.05	12000	1000	1	500	111
Simulation_205	2.666	0.056	0.178	0.031	2.23	0.906	205	0.05	12000	1000	1	500	111
Simulation_206	2.163	0.033	0.114	0.141	2.858	0.227	206	0.05	12000	1000	1	500	111
Simulation_207	1.157	0.079	0.242	0.068	1.603	0.68	207	0.05	12000	1000	1	500	111
Simulation_208	1.22	0.036	0.122	0.054	0.583	0.935	208	0.05	12000	1000	1	500	111
Simulation_209	2.226	0.081	0.247	0.127	1.838	0.482	209	0.05	12000	1000	1	500	111
Simulation_210	2.729	0.014	0.061	0.018	2.466	0.708	210	0.05	12000	1000	1	500	111
Simulation_211	1.723	0.059	0.186	0.091	1.211	0.256	211	0.05	12000	1000	1	500	111
Simulation_212	1.974	0.025	0.092	0.072	2.779	0.142	212	0.05	12000	1000	1	500	111
Simulation_213	2.98	0.07	0.217	0.145	1.525	0.595	213	0.05	12000	1000	1	500	111
Simulation_214	2.477	0.047	0.153	0.036	0.897	0.369	214	0.05	12000	1000	1	500	111
Simulation_215	1.472	0.093	0.281	0.109	2.152	0.821	215	0.05	12000	1000	1	500	111
Simulation_216	1.346	0.019	0.075	0.136	1.995	0.878	216	0.05	12000	1000	1	500	111
Simulation_217	2.352	0.064	0.2	0.063	0.74	0.425	217	0.05	12000	1000	1	500	111
Simulation_218	2.855	0.042	0.139	0.1	1.368	0.652	218	0.05	12000	1000	1	500	111
Simulation_219	1.849	0.087	0.264	0.027	2.623	0.199	219	0.05	12000	1000	1	500	111
Simulation_220	1.597	0.053	0.169	0.118	1.681	0.312	220	0.05	12000	1000	1	500	111
Simulation_221	2.603	0.098	0.294	0.045	2.936	0.765	221	0.05	12000	1000	1	500	111
Simulation_222	2.1	0.031	0.108	0.081	2.309	0.539	222	0.05	12000	1000	1	500	111
Simulation_223	1.094	0.076	0.233	0.009	1.054	0.991	223	0.05	12000	1000	1	500	111
Simulation_224	1.063	0.052	0.167	0.102	2.897	0.157	224	0.05	12000	1000	1	500	111
Simulation_225	2.069	0.097	0.292	0.029	1.642	0.609	225	0.05	12000	1000	1	500	111
Simulation_226	2.572	0.029	0.103	0.138	1.015	0.383	226	0.05	12000	1000	1	500	111
Simulation_227	1.566	0.074	0.228	0.066	2.27	0.836	227	0.05	12000	1000	1	500	111
Simulation_228	1.817	0.018	0.072	0.084	0.701	0.949	228	0.05	12000	1000	1	500	111
Simulation_229	2.823	0.063	0.197	0.011	1.956	0.496	229	0.05	12000	1000	1	500	111
Simulation_230	2.32	0.04	0.133	0.12	2.583	0.722	230	0.05	12000	1000	1	500	111
Simulation_231	1.314	0.086	0.261	0.047	1.328	0.27	231	0.05	12000	1000	1	500	111
Simulation_232	1.44	0.023	0.086	0.02	1.485	0.213	232	0.05	12000	1000	1	500	111
Simulation_233	2.446	0.069	0.214	0.093	2.74	0.666	233	0.05	12000	1000	1	500	111
Simulation_234	2.949	0.046	0.15	0.056	2.113	0.439	234	0.05	12000	1000	1	500	111
Simulation_235	1.943	0.091	0.275	0.129	0.858	0.892	235	0.05	12000	1000	1	500	111
Simulation_236	1.692	0.035	0.119	0.038	1.799	0.779	236	0.05	12000	1000	1	500	111
Simulation_237	2.697	0.08	0.244	0.111	0.544	0.326	237	0.05	12000	1000	1	500	111
Simulation_238	2.194	0.012	0.056	0.075	1.172	0.553	238	0.05	12000	1000	1	500	111
Simulation_239	1.189	0.057	0.181	0.147	2.426	0.1	239	0.05	12000	1000	1	500	111
Simulation_240	1.126	0.015	0.064	0.116	2.348	0.355	240	0.05	12000	1000	1	500	111
Simulation_241	2.132	0.06	0.189	0.043	1.093	0.807	241	0.05	12000	1000	1	500	111
Simulation_242	2.635	0.038	0.128	0.079	1.721	0.128	242	0.05	12000	1000	1	500	111
Simulation_243	1.629	0.083	0.253	0.006	2.975	0.581	243	0.05	12000	1000	1	500	111
Simulation_244	1.88	0.049	0.158	0.134	1.407	0.694	244	0.05	12000	1000	1	500	111
Simulation_245	2.886	0.094	0.283	0.061	2.662	0.241	245	0.05	12000	1000	1	500	111
Simulation_246	2.383	0.026	0.094	0.097	2.034	0.92	246	0.05	12000	1000	1	500	111
Simulation_247	1.377	0.072	0.222	0.025	0.779	0.468	247	0.05	12000	1000	1	500	111
Simulation_248	1.251	0.043	0.142	0.07	0.936	0.524	248	0.05	12000	1000	1	500	111
Simulation_249	2.257	0.089	0.269	0.143	2.191	0.977	249	0.05	12000	1000	1	500	111
Simulation_250	2.76	0.021	0.081	0.034	2.819	0.298	250	0.05	12000	1000	1	500	111
Simulation_251	1.754	0.066	0.206	0.106	1.564	0.751	251	0.05	12000	1000	1	500	111
Simulation_252	1.503	0.032	0.111	0.052	2.505	0.638	252	0.05	12000	1000	1	500	111
Simulation_253	2.509	0.077	0.236	0.125	1.25	0.185	253	0.05	12000	1000	1	500	111
Simulation_254	2.006	0.055	0.175	0.016	0.623	0.864	254	0.05	12000	1000	1	500	111
Simulation_255	1	0.1	0.3	0.088	1.877	0.411	255	0.05	12000	1000	1	500	111
Simulation_256	1.004	0.055	0.175	0.046	2.588	0.979	256	0.05	12000	1000	1	500	111
Simulation_257	2.01	0.1	0.3	0.119	1.333	0.526	257	0.05	12000	1000	1	500	111
Simulation_258	2.513	0.032	0.111	0.01	0.706	0.752	258	0.05	12000	1000	1	500	111
Simulation_259	1.507	0.077	0.236	0.082	1.961	0.3	259	0.05	12000	1000	1	500	111
Simulation_260	1.758	0.021	0.081	0.064	1.02	0.187	260	0.05	12000	1000	1	500	111
Simulation_261	2.764	0.066	0.206	0.137	2.275	0.639	261	0.05	12000	1000	1	500	111
Simulation_262	2.261	0.043	0.142	0.028	2.902	0.413	262	0.05	12000	1000	1	500	111
Simulation_263	1.255	0.089	0.269	0.101	1.647	0.866	263	0.05	12000	1000	1	500	111
Simulation_264	1.381	0.026	0.094	0.146	1.176	0.809	264	0.05	12000	1000	1	500	111
Simulation_265	2.387	0.072	0.222	0.073	2.431	0.356	265	0.05	12000	1000	1	500	111
Simulation_266	2.89	0.049	0.158	0.11	1.804	0.583	266	0.05	12000	1000	1	500	111
Simulation_267	1.884	0.094	0.283	0.037	0.549	0.13	267	0.05	12000	1000	1	500	111
Simulation_268	1.633	0.038	0.128	0.128	2.118	0.243	268	0.05	12000	1000	1	500	111
Simulation_269	2.639	0.083	0.253	0.055	0.863	0.696	269	0.05	12000	1000	1	500	111
Simulation_270	2.136	0.015	0.064	0.091	1.49	0.47	270	0.05	12000	1000	1	500	111
Simulation_271	1.13	0.06	0.189	0.019	2.745	0.922	271	0.05	12000	1000	1	500	111
Simulation_272	1.193	0.012	0.056	0.023	2.039	0.668	272	0.05	12000	1000	1	500	111
Simulation_273	2.198	0.058	0.183	0.096	0.784	0.215	273	0.05	12000	1000	1	500	111
Simulation_274	2.701	0.035	0.119	0.06	1.412	0.894	274	0.05	12000	1000	1	500	111
Simulation_275	1.695	0.08	0.244	0.132	2.667	0.441	275	0.05	12000	1000	1	500	111
Simulation_276	1.947	0.046	0.15	0.005	1.725	0.328	276	0.05	12000	1000	1	500	111
Simulation_277	2.953	0.092	0.278	0.078	2.98	0.781	277	0.05	12000	1000	1	500	111
Simulation_278	2.45	0.024	0.089	0.041	2.353	0.102	278	0.05	12000	1000	1	500	111
Simulation_279	1.444	0.069	0.214	0.114	1.098	0.554	279	0.05	12000	1000	1	500	111
Simulation_280	1.318	0.041	0.136	0.087	0.627	0.611	280	0.05	12000	1000	1	500	111
Simulation_281	2.324	0.086	0.261	0.014	1.882	0.158	281	0.05	12000	1000	1	500	111
Simulation_282	2.827	0.018	0.072	0.123	2.51	0.837	282	0.05	12000	1000	1	500	111

Results/M0/params.R
# Header -------------------------------------------------------------
# Fixed parameters across species for the M0 model (Precipitations and
# Temperature constraints)
# Value range from Skeels et al. (2023)

# Parameters ---------------------------------------------------------
params <- list(
  # Random seed
  seed = 123,
  # Max total nb. of species
  max_species = 12000,
  # Max nb. of species per cell
  max_coex_sp = 300,
  # Number of species to start with
  start_species = 1,
  # Divergence threshold above which speciation occurs
  S = runif(1, min = 0.75, max = 1.25),
  # Temperature and precipitation niche breadth
  omega_T = runif(1, 0.01, 0.076), omega_P = runif(1, 0.05, 0.125),
  # Rate of environment niche evolution
  sigma_e = runif(1, 0.0075, 0.0225),
  # Shape and scale of the dispersal kernel (Weibull)
  disp_shape = runif(1, 1, 3), disp_scale = runif(1, 0.25, 2))



# library(randtoolbox)
# 
# a2 <- sobol(n=500, dim = 2)
# omega <- data.frame(om_t = runif(n = 500, min = 0.01, max = 0.076),
#                     om_p = runif(n = 500, min = 0.05, max = 0.125))
# par(mfrow = c(1,2))
# plot(x = omega[,1], y = omega[,2], 
#      xlab = "temp", ylab = "prec", main = "Uniform")
# plot(a2[,1]*(0.076-0.01)+0.01,
#      a2[,2]*(0.126-0.05)+0.05,
#      xlab = "temp", ylab = "prec", main = "Sobol")
